pkg/monitor/agent.go

```diff
@@ -553,20 +553,24 @@ func (monitor *AgentMonitor) fetchPortLocked(uuid ovsdb.UUID, bridgeName string)
 	}
 
 	for _, uuid := range listUUID(ovsPort.Fields["interfaces"]) {
-		iface, err := monitor.fetchInterfaceLocked(uuid, bridgeName)
-		if err != nil {
-			return nil, err
+		iface := monitor.fetchInterfaceLocked(uuid, bridgeName)
+		if iface != nil {
+			port.Interfaces = append(port.Interfaces, *iface)
 		}
-		port.Interfaces = append(port.Interfaces, *iface)
 	}
 
 	return port, nil
 }
 
-func (monitor *AgentMonitor) fetchInterfaceLocked(uuid ovsdb.UUID, bridgeName string) (*agentv1alpha1.OVSInterface, error) {
+func (monitor *AgentMonitor) fetchInterfaceLocked(uuid ovsdb.UUID, bridgeName string) *agentv1alpha1.OVSInterface {
 	ovsIface, ok := monitor.ovsdbCache["Interface"][uuid.GoUuid]
 	if !ok {
```

Comment:
I think we need to add some logs here too.

Reply:
done